apps/condo/domains/user/components/UserOrganizationsList.tsx

```diff
@@ -48,6 +48,7 @@ interface IOrganizationEmployeeItem {
 const OrganizationEmployeeItem: React.FC<IOrganizationEmployeeItem> = (props) => {
     const intl = useIntl()
     const OrganizationMessage = intl.formatMessage({ id: 'pages.condo.property.field.Organization' })
+    const InnMessage = intl.formatMessage({ id: 'pages.condo.property.field.Inn'})
```

Comment:
It would be better to call it `tin` :)
```suggestion
    const TinMessage = intl.formatMessage({ id: 'pages.condo.property.field.Tin' })
```